app/controlplane/plugins/core/guac/v1/README.md

```diff
@@ -0,0 +1,96 @@
+# GUAC integration
+
+Graph for Understanding Artifact Composition ([GUAC](https://github.com/guacsec/guac)) aggregates software security metadata into a high fidelity graph database—normalizing entity identities and mapping standard relationships between them. Querying this graph can drive higher-level organizational outcomes such as audit, policy, risk management, and even developer assistance.
+
+This integration allows Chainloop users to automatically send attestation (DSSE envelopes / in-toto statements) and CycloneDX/SPDX Software Bill Of Materials (SBOMs) to a cloud storage bucket staging area. From there, GUAC can be configured to continuously monitor and inject that data.
```

Comment:
Should we add the supported input formats section? Could our readme generator do it? I think we have a few on the list to add sooner than later in chainloop, like scorecards: https://github.com/guacsec/guac/tree/main/pkg/handler/processor

Reply:
Good idea! Maybe as part of this? https://github.com/chainloop-dev/chainloop/issues/193

If so. Would you mind adding a comment there?

Thanks!